.travis.yml

```diff
@@ -26,5 +26,4 @@ notifications:
 matrix:
   allow_failures:
     - env: SYMFONY_VERSION=dev-master
```

Comment:
I don't see the interest of keeping this in the allow failures section as almost no one look at them.
I prefer that the tests break and that we know that we will have to change something.